### PR TITLE
Display length of shortest path in local results UI

### PR DESCRIPTION
**Pull request by p-**
Hey 👋

This PR implements the my feature request from https://github.com/github/vscode-codeql/issues/3659 for local path queries.
I'm totally open for changes of course (e.g. visualize it differently etc.)

The changes currently look like this:
display-shortest-path-local

(a badge showing the shortest path (`4` or `12` here in this screenshot) was added and the length of the paths is shown on the respective entries in parentheses (e.g. `(12 steps)`). Additionally, a tooltip `Shortest path` is shown when hovering over the badge)

**Note**:

* most of the added lines are for tests which I newly created. (the actual changes are quite small)
* I've copied the contents for `mockresults.ts` from this [story](https://github.com/github/vscode-codeql/blob/96c33a14b8662b28ec19f648381905177cdc9cea/extensions/ql-vscode/src/stories/results/AlertTable.stories.tsx#L324).

### CODE REVIEW

File: extensions/ql-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## [UNRELEASED]
 
+- Update results view to display the length of the shortest path for path queries.
```
Comment (aeisenberg) [inline extensions/ql-vscode/CHANGELOG.md:5]:
```suggestion
- Update results view to display the length of the shortest path for path queries. [#3687](https://github.com/github/vscode-codeql/pull/3687)
```

**Review by aeisenberg — COMMENTED** [review]
Nice. Thanks for doing this. One minor change to the changelog entry and then 👍🏼.

File: extensions/ql-vscode/src/view/results/__tests__/AlertTableResultRow.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { render as reactRender, screen } from "@testing-library/react";
+import { AlertTableResultRow } from "../AlertTableResultRow";
+import type { Props } from "../AlertTablePathRow";
```
Comment (p-) [inline extensions/ql-vscode/src/view/results/__tests__/AlertTableResultRow.spec.tsx:3]:
```suggestion
import type { Props } from "../AlertTableResultRow";
```

Comment (p-) [inline extensions/ql-vscode/src/view/results/__tests__/AlertTableResultRow.spec.tsx:3]:
```suggestion
import type { Props } from "../AlertTableResultRow";
```

**Comment by p-** [comment]
@aeisenberg wow thanks!

**Review by aeisenberg — APPROVED** [review]
🚢

**Pull request merged**